src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/RoundTripIslSpec.groovy

```diff
@@ -293,4 +294,60 @@ round trip latency rule is removed on the dst switch"() {
             }
         }
     }
+
+    @Tidy
+    @Tags([SMOKE_SWITCHES])
+    def "A round-trip latency non-direct ISL goes UP according to antiflap"() {
+        given: "An active round-trip a-switch link"
+        def isl = topology.islsForActiveSwitches.find { it.aswitch?.inPort && it.aswitch?.outPort &&
+                [it.srcSwitch, it.dstSwitch].every { it.features.contains(SwitchFeature.NOVIFLOW_COPY_FIELD) }
+        }
+        assumeTrue("Wasn't able to find round-trip ISL with a-switch", isl != null)
+
+        when: "Port down event happens"
+        def timestampBefore = System.currentTimeMillis()
+        northbound.portDown(isl.srcSwitch.dpId, isl.srcPort)
+        def portIsDown = true
+        Wrappers.wait(WAIT_OFFSET) {
+            assert northbound.getLink(isl).state == FAILED
+            assert northbound.getLink(isl.reversed).state == FAILED
+        }
+
+        and: "Port up event happens"
+        northbound.portUp(isl.srcSwitch.dpId, isl.srcPort)
+        portIsDown = false
+
+        then: "The ISL is failed till 'antiflap' is deactivated"
```

Comment:
Overall and according to this step, it more feels that you are testing antiflap, rather than rtl. So I'd rather it was in antiflap spec

Reply:
moded into `PortAntiflapSpec`